### PR TITLE
Content folders

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

**Comment by fbertsch** [comment]
Fixes https://github.com/mozilla/lookml-generator/issues/36.

Would have loved to use terraform instead.

File: generator/content.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+    shared_folders = sdk.search_folders(name="home")
+    if len(shared_folders) != 1:
+        raise click.ClickException(
+            f"Found wrong number of shared folders: {shared_folders}"
+        )
+
+    shared_folder_id = shared_folders[0].id
+    if shared_folder_id is None:
+        raise click.ClickException("Error: Shared folder id is missing")
```
Comment (relud) [inline generator/content.py:36]: this pattern is used so many times, I think it should be a separate function that is passed a list and returns an id

**Review by relud — APPROVED** [review]
r+wc

File: generator/content.py (hunk 1 above)
Comment (fbertsch) [reply generator/content.py:36]: Makes sense, thanks for the suggestion!

**Pull request merged**